types/cache/buildcache/types.go

```diff
@@ -0,0 +1,8 @@
+package buildcache
+
+type (
+	Metadata struct {
+		TotalTasks int `json:"total_tasks"`
+		Cached     int `json:"cached"`
```

Comment:
Either make a Tasks struct or make both `total_tasks` and `cached_tasks`

Reply:
made it cached_tasks